bc_obps/registration/migrations/0079_auto_20250213_2335.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 5.0.11 on 2025-02-13 23:35
+
+from django.db import migrations
+
+"""
+One-time forward-only migration to be applied to prod data.
+Purpose: Remove the point_of_contact (created in Reg1) from the `operation` model and instead relate this contact to operations via contact.operator and the operation_contacts model.
+"""
+
+
+def migrate_point_of_contact(apps, schema_monitor):
+    '''
+    Migrate operation.point_of_contact to operation_contacts and contact.operator
+    '''
+    Operation = apps.get_model('registration', 'Operation')
+
+    operations = Operation.objects.all()
+
+    for operation in operations:
+        if operation.point_of_contact:
+            operation.contacts.add(operation.point_of_contact)
+            operation.operator.contacts.add(operation.point_of_contact)
```

Comment:
The related manager does come in OK with the setup on line 15. The error we were seeing in the pairing session was on the asserts because I forgot to put `.all()` initially